src/server.js

```diff
@@ -41,6 +45,9 @@ router.post('/login', usersController.login);
 /* Start server */
 app.use('/', router);
 
+// Configure mySQL DB
+app.set('db', db);
```

Comment:
is this where we would change it to devDb if we're using the dev environment?

Reply:
Yes! For our current settings we can manually change it for development, or if we use NODE_ENV to set up different environments we can also define the related db here